app/src/main/java/com/pileproject/drive/setting/app/ProgramListFragment.java

```diff
@@ -80,17 +92,25 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         mProgramsCheckAllButton[0] = (Button) v.findViewById(R.id.programList_samplePrograms_checkAllButton);
         mProgramsCheckAllButton[1] = (Button) v.findViewById(R.id.programList_userPrograms_checkAllButton);
 
+
         return v;
     }
 
     @Override
     public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
 
+        final ProgramListFragment thiz = this;
```

Comment:
It can't because AlertDialogFragment.Builuder needs the parent fragment instance rather than a context in this case.
I think another resort is maintaining 'this' as a field variable, but it gets uglier I think...